### PR TITLE
Procedures for funding leads and snafus

### PR DESCRIPTION
**Pull request by adina**
Also includes update on subcommittee that never got added in the past.

### CODE REVIEW

File: handling-funding-leads.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#Handling Funding Leads
+
+Funding leads should be communicated by email directly to the committee head of the Partner Relations / Development Committee (and copied to the Executive Director and Steering Committee) with the term “Funding Lead” in the subject line. The Partner Relations / Development Committee will follow leads and with a protocol dealing with template emails for explaining partnership, how to handle requests for customization, who handles communication with those partners, how to set up the contract, and how to request invoices from NumFOCUS.
+
```
Comment (karinlag) [inline handling-funding-leads.md]: This means that the PRD committee makes a procedure for who does what and when?

**Comment by rgaiacs** [comment]
@karinlag Do you remember if this was approved?

**Comment by karinlag** [comment]
I think it was sort of approved, but never acted on. Now that we don't have that committee (at least not right now) this is kind of moot.

**Comment by jduckles** [comment]
Any objection to closing this?

**Comment by jduckles** [comment]
Closing as outdated.

**Pull request closed without merging**